src/InflateStream.php

```diff
@@ -20,10 +20,35 @@ class InflateStream implements StreamInterface
 
     public function __construct(StreamInterface $stream)
     {
-        // Skip the first 10 bytes
-        $stream = new LimitStream($stream, -1, 10);
+        // read the first 10 bytes, ie. gzip header
+        $header = $stream->read(10);
+        $filename_header_length = $this->getLengthOfPossibleFilenameHeader($stream, $header);
```

Comment:
Can you update this variable to be camelCased so it matches the rest of the code base?